### PR TITLE
develop <- feature/69_logging : Adding Log4j2 for logging

### PR DESCRIPTION
**Pull request by shulolo**

### CODE REVIEW

File: src/main/java/com/dakar/dakar/controller/GraphQlController.java [hunk 1]
```diff
@@ -27,6 +29,9 @@
     @Autowired
     private GraphQL graphQL;
 
+    // Logger
+    private static final Logger logger = LogManager.getLogger(GraphQlController.class);
```
Comment (blablanumerodeux) [inline src/main/java/com/dakar/dakar/controller/GraphQlController.java]: this should be done automatically by the @Slf4j annotation
If it's not the case, then we have a problem

File: src/main/java/com/dakar/dakar/controller/GraphQlController.java [hunk 2]
```diff
@@ -35,6 +40,7 @@
      */
     @Bean
     RouterFunction<ServerResponse> routesGraphQl() {
+        logger.info("Graphql route");
```
Comment (blablanumerodeux) [inline src/main/java/com/dakar/dakar/controller/GraphQlController.java]:
```suggestion
        log.info("Graphql route");
```

File: src/main/resources/log4j2.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <ThresholdFilter level="ERROR" onMatch="ACCEPT" onMismatch="DENY"/>
+            <PatternLayout>
+                <Pattern>${LOG_PATTERN}</Pattern>
+            </PatternLayout>
+        </SMTP>
+         -->
+    </Appenders>
+
+    <Loggers>
+        <AsyncLogger name="comm.dakar.dakar" level="debug"
```
Comment (blablanumerodeux) [inline src/main/resources/log4j2.xml]:
```suggestion
        <AsyncLogger name="com.dakar.dakar" level="debug"
```

Comment (blablanumerodeux) [reply src/main/resources/log4j2.xml]: one line please

**Review by blablanumerodeux — CHANGES_REQUESTED** [review]

File: src/main/java/com/dakar/dakar/controller/GraphQlController.java (hunk 1 above)
Comment (shulolo) [reply src/main/java/com/dakar/dakar/controller/GraphQlController.java]: you are right

File: src/main/java/com/dakar/dakar/controller/ReactiveController.java [hunk 4]
```diff
@@ -106,7 +112,7 @@
      */
     @Bean
     RouterFunction<ServerResponse> routeForCouch() {
-        String id = UUID.randomUUID().toString();
```
Comment (blablanumerodeux) [inline src/main/java/com/dakar/dakar/controller/ReactiveController.java]: why ?

**Review by blablanumerodeux — DISMISSED** [review]

**Review by blablanumerodeux — APPROVED** [review]

**Pull request merged**